cryostatio/cryostat-helm
# feat(scheduling): per-Deployment k8s scheduling options

Related to #220

To test:
1. `helm install cryostat ./charts/cryostat` and ensure everything comes up as usual
2. If you have a multi-node cluster for testing, try setting different nodeSelector, tolerations, and affinity settings introduced by this PR. Else, just verify that the k8s resource manifests look as expected.
